src/headers/tomcrypt_math.h

```diff
@@ -339,10 +355,15 @@ typedef struct {
        @param G   The point to multiply
        @param R   The destination for kG
        @param modulus  The modulus for the field
-       @param map Boolean indicated whether to map back to affine or not (can be ignored if you work in affine only)
+       @param map Boolean indicated whether to map back to affine or not
+                  (can be ignored if you work in affine only)
        @return CRYPT_OK on success
    */
-   int (*ecc_ptmul)(void *k, ecc_point *G, ecc_point *R, void *modulus, int map);
+   int (*ecc_ptmul)(     void *k,
```

Comment:
Simply rebase your stuff on develop (before merging this) and I'll take care of a rebase of `miko-ecc-enhancements` after merging this.